upload-s3-assets/action.yml

```diff
@@ -25,5 +25,9 @@ runs:
         if [ "${{ inputs.dry_run }}" == "false" ]; then
           for filename in ${{ inputs.filenames }}; do aws s3 cp ${filename} s3://${AWS_BUCKET}/${{ inputs.product_name }}/${{ inputs.version }}/${filename}; done
         else
-          for filename in ${{ inputs.filenames }}; do echo "Would upload ${filename} to ${{ inputs.product_name }}/${{ inputs.version }}/${filename}"; done
+          for filename in ${{ inputs.filenames }}; do
+            echo "Would upload ${filename} to ${{ inputs.product_name }}/${{ inputs.version }}/${filename}";
+            echo "Contents: "
+            cat ${filename}
```

Comment:
We've generally tried to avoid printing file contents as it may contain information that is not supposed to be public (but not exactly sensitive either). Any opinion @blink1073?

Reply:
I can revert this, but I found that without printing the contents of the files it was impossible to determine whether or not the action was working properly without uploading the files to S3 (which makes the dry_run logic here irrelevant).  This would be useful to have while I incorporate the action into Node's other repos, but if we don't want to print I can just upload files to s3 to confirm things are working.

Reply:
I think the only file that is somewhat sensitive currently is the SARIF file, but I agree it would be a bad decision to leave this here for all files.

Reply:
You could instead print something like the stats of the file.